<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModDetectorDialog</class>
 <widget class="QDialog" name="ModDetectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改探测器信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>32</y>
     <width>251</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>探测器编号*   ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtDetectorNo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成，固定长3位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>97</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>探测器信道编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtDetectorChannelID"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color:red</string>
       </property>
       <property name="text">
        <string>由0-9的数字组成</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>备          注：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtDetectorComments"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>是否重启探测器*：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbReset">
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>阀    值：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbxThreshold"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>延长时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbxDelayTime"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptnMod">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
